frontend/src/components/Popup.tsx

```diff
@@ -2,14 +2,14 @@ import { motion } from "framer-motion";
 import { useEffect } from "react";
 
 
-interface ModalInterface {
+interface Popup {
```

Comment:
Here you have both an interface called Popup and a function called Popup.
That means there's a name collision. I would call this `PopupProps`

Reply:
done!